cluster/v1/0000_00_clusters.open-cluster-management.io_managedclusters.crd.yaml

```diff
@@ -3,13 +3,29 @@ kind: CustomResourceDefinition
 metadata:
   name: managedclusters.cluster.open-cluster-management.io
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .spec.hubAcceptsClient
+    name: Hub Accepted
+    type: boolean
+  - JSONPath: .spec.managedClusterClientConfigs[*].url
```

Comment:
(I believe `additionalPrinterColumns` is an ordered field that dictates the ordering of these fields in CLI, if not, please correct me).

We should think carefully about the ordering of these fields; I would probably have the condition-based fields, then this field, then age.